Dockerfile-test

```diff
@@ -1,42 +1,24 @@
-FROM python:2.7.13-alpine3.6
-
-# install a nice prompt
-RUN apk --no-cache --update add bash \
-	&& rm -rf /var/cache/apk/*
-
-# install Solidity compiler (solc)
-RUN apk --no-cache --update add build-base cmake boost-dev git ca-certificates wget openssl \
-	&& update-ca-certificates \
-	&& sed -i -E -e 's/include <sys\/poll.h>/include <poll.h>/' /usr/include/boost/asio/detail/socket_types.hpp \
-	&& mkdir /solidity \
-	&& wget -qO- https://github.com/ethereum/solidity/releases/download/v0.4.13/solidity_0.4.13.tar.gz | tar -xvz -C /solidity --strip 1 \
-	&& cd /solidity && cmake -DCMAKE_BUILD_TYPE=Release -DTESTS=0 -DSTATIC_LINKING=1 \
-	&& cd /solidity && make solc && install -s  solc/solc /usr/bin \
-	&& cd / && rm -rf solidity \
-	&& apk del sed build-base git make cmake gcc g++ musl-dev curl-dev boost-dev \
-	&& rm -rf /var/cache/apk/*
+FROM python:2.7.13
+
+RUN wget --quiet --output-document /usr/local/bin/solc \
+  https://github.com/ethereum/solidity/releases/download/v0.4.13/solc-static-linux && \
```

Comment:
Also recommend keeping each command on one line, not split across lines.  In this case, this line is part of the previous line so there should not be a line break here.